ntoskrnl/io/iomgr/irq.c

```diff
@@ -166,4 +166,67 @@ IoDisconnectInterrupt(PKINTERRUPT InterruptObject)
     ExFreePool(IoInterrupt); // ExFreePoolWithTag(IoInterrupt, TAG_KINTERRUPT);
 }
 
+#if (NTDDI_VERSION >= NTDDI_LONGHORN) || defined(__REACTOS__)
+NTSTATUS
+NTAPI
+IopConnectInterruptEx_FullySpecific(_Inout_ PIO_CONNECT_INTERRUPT_PARAMETERS Parameters)
```

Comment:
Underscore is not usually used in names. I suggest `IopConnectInterruptFullySpecific`